src/kernel/kshell.c

```diff
@@ -294,6 +294,12 @@ int try_exec(const char* command) {
     return 0;
 }
 
+int overflowtest() {
```

Comment:
We could maybe move this to `selftest` instead, what do you think?

Reply:
Not a good idea since it'll cause the kernel to panic due to the stack smash protector catching the overflow

Reply:
ha, that's right.